src/components/SkeletonBodyText/SkeletonBodyText.vue

```diff
@@ -0,0 +1,37 @@
+<template lang="pug">
+AppProvider
+  div(
+    v-for="index in 5"
+    :class="styles.SkeletonBodyText",
+  )
+</template>
+
+<script setup lang="ts">
+import styles from '@/classes/Demo.json'
+import {computed} from "vue";
+import {AppProvider} from "@/components";
+
+interface Props {
+  /**
+   * Number of lines to display
+   * @default 3
+   */
+  lines?: number;
+}
+
+const props = defineProps<Props>();
+
+const lines = computed(()=> {
```

Comment:
`lines` can't be computed because it is `prop`. And actually no need this `computed`